package.json

```diff
@@ -64,13 +64,22 @@
   "license": "BSD-3-Clause",
   "oclif": {
     "commands": "./lib/commands",
-    "bin": "sfdx",
+    "hooks": {
+      "plugins:preinstall:verify:signature": [
+        "./dist/hooks/verifyInstallSignature.js"
+      ]
+    },
     "devPlugins": [
       "@oclif/plugin-help"
     ],
     "topics": {
-      "hello": {
-        "description": "Commands to say hello."
+      "plugins": {
+        "description": "commands to verify the authenticity of a plugin",
```

Comment:
This is not the description for the plugins topic. Remove, it is defined elsewhere.